src/main.c

```diff
@@ -275,7 +276,37 @@ static void MX_TIM3_Init(void)
   */
 static void MX_GPIO_Init(void)
 {
-  // TODO - implement GPIO initializations here...
+
+  GPIO_InitTypeDef GPIO_InitStruct = {0};
+
+  /* GPIO Ports Clock Enable */
+  __HAL_RCC_GPIOA_CLK_ENABLE();
+  __HAL_RCC_GPIOB_CLK_ENABLE();
+
+  /* Set default GPIO pin Output Level */
+  HAL_GPIO_WritePin(INSP_VALVE_PORT, INSP_VALVE_PIN, GPIO_PIN_RESET);
+  HAL_GPIO_WritePin(EXP_VALVE_PORT, EXP_VALVE_PIN, GPIO_PIN_RESET);
+  HAL_GPIO_WritePin(TANK_VALVE_PORT, TANK_VALVE_PIN, GPIO_PIN_RESET);
+
+  /* Configure each GPIO pin */
+  GPIO_InitStruct.Pin = INSP_VALVE_PIN;
+  GPIO_InitStruct.Mode = GPIO_MODE_OUTPUT_PP;
+  GPIO_InitStruct.Pull = GPIO_NOPULL;
+  GPIO_InitStruct.Speed = GPIO_SPEED_LOW;
```

Comment:
Is it possible for someone from the platform team to look at this line. 

What is the difference between SPEED_LOW  and SPEED_VERY_HIGH ?

Why choose one instead of the other ?

Reply:
From the comments in the HAL... 

![image](https://user-images.githubusercontent.com/1484030/97645871-52243880-1a0b-11eb-95be-0a8512bc38a7.png)

Note that `GPIO_SPEED_LOW` is mapped to `GPIO_SPEED_FREQ_LOW`. 

I don't think we need to be toggling this GPIO at faster that 2 MHz. So speed low is OK here.

Reply:
We **definitely** won't need to be toggling these pins faster than 2 MHz. 

The fastest we likely need to toggle these pins (valves, leds) would be about 5 orders of magnitude slower... 20ms or so.

Reply:
Good to double check it though!